develop/interoperability/intro-to-xcm.md

```diff
@@ -1,4 +1,110 @@
+
+* **Asymmetric**
+
+    XCM messages, by default, do not have results that let the sender know that the message was received - they follow the 'fire and forget' paradigm. Any results must be separately communicated to the sender with an additional message back to the origin.
+
+* **Agnostic**
+
+    XCM makes no assumptions about the nature of the consensus systems between which the messages are being passed. XCM as a message format should be usable in any system that derives finality through consensus.
+
+These four crucial design decisions allow for XCM messages to be a reliable yet convenient way to properly convey the intentions from one consensus system to another without any compatibility issues.
+
+## The XCM Tech Stack
+
+<figure markdown>
+  <img src="../../../images/develop/parachain-devs/interoperability/xcm-tech-stack/cross-consensus-tech-stack-e9c1b2ab8b6f6f3f9a78b3a412af0698.png" width="800">
+  <figcaption>The XCM Tech Stack</figcaption>
+</figure>
+
+## Core Functionalities of XCM
+Some of the key features and additions to cross-consensus messaging include
+
+* **Programmability**
+
+    The ability to have expectations for messages, which allow for more comprehensive use cases, safe dispatches for version checking, branching, and NFT/Asset support.
+
+* **Functional Multichain Decomposition**
+
+    The ability to define mechanisms to cross-reference and perform actions on other chains on behalf of the origin chain (remote locking), context/id for these messages, and asset namespacing.
+
+* **Bridging**
+
+    Introduces the concept of a universal location, which allows for a base reference for global consensus systems for multi-hop setups. This location is above the parent relay chain or other consensus systems like Ethereum or Bitcoin.
+
+A core part of the vision that XCM provides is improving communication between the chains to make system parachains a reality. For example, the Polkadot relay chain handles more than just parachain management and shared security - it handles user balances/assets, governance, and staking. Ideally, the relay chain should be for what it's intended to be - a place for shared security. System parachains can alleviate these core responsibilities from the relay chain but only by using a standard format like XCM.
+
+This is where system parachains come in, where each of these core responsibilities can be delegated to a system parachain respectively.
+
+## XCM Example
+Below shows a simple example of an XCM message. In the example, the native token is withdrawn from the account of Alice and deposited into the account of Bob. The message simulates a transfer between the two accoubts in the same consensus system (ParaA).  You can find the complete example in the [repo](https://github.com/paritytech/xcm-docs/tree/main/examples){target=_blank}.
```

Comment:
This is a nitpick comment, but maybe it would be nice to create a note below this rust snippet with this sentence: "You can find the complete example in..."

```suggestion
## XCM Example

A simple example of an XCM message is shown below. In the example, the native token is withdrawn from Alice's account and deposited into Bob's account. The message simulates a transfer between the two accounts in the same consensus system (`ParaA`). You can find the complete example in the [XCM Docs](https://github.com/paritytech/xcm-docs/tree/main/examples){target=\_blank} repo.
```

Reply:
Also, it would make sense to add a direct reference to the Withdraw instruction - https://github.com/polkadot-fellows/xcm-format?tab=readme-ov-file#withdrawasset

Reply:
Added!